build-system/tasks/make-extension/bento/index.js

```diff
@@ -1,166 +0,0 @@
-/**
```

Comment:
This file is now largely `build-system/tasks/make-extension/index.js` below, but was not seen as renamed by Git.